quantum/keycode_config.c

```diff
@@ -16,7 +16,9 @@
 
 #include "keycode_config.h"
 
-extern keymap_config_t keymap_config;
+#ifdef KEYMAP_CONFIG_RAM
+keymap_config_t keymap_config;
```

Comment:
If the magic.c declaration isn't changed, this will error out on complication, I think.